### PR TITLE
layers: Better timeline semaphore error messages

### PR DESCRIPTION
**Pull request by sfricke-samsung**
Was playing around with VK_KHR_timeline_semaphore and realized some of the error messages were less than desired to figure out the issue. Fixed up the vkQueueSubmit and vkQueueBindSparse messages to be more useful

### CODE REVIEW

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -2616,29 +2616,32 @@ bool CoreChecks::ValidateSemaphoresForSubmit(VkQueue queue, const VkSubmitInfo *
         VkSemaphore semaphore = submit->pWaitSemaphores[i];
         const auto *pSemaphore = GetSemaphoreState(semaphore);
         if (pSemaphore && pSemaphore->type == VK_SEMAPHORE_TYPE_TIMELINE_KHR && !timeline_semaphore_submit_info) {
-            skip |= LogError(semaphore, "VUID-VkSubmitInfo-pWaitSemaphores-03239",
-                             "VkQueueSubmit: %s is a timeline semaphore, but pBindInfo does not"
-                             "include an instance of VkTimelineSemaphoreSubmitInfoKHR",
-                             report_data->FormatHandle(semaphore).c_str());
+            skip |=
+                LogError(semaphore, "VUID-VkSubmitInfo-pWaitSemaphores-03239",
+                         "VkQueueSubmit: pSubmits[%u].pWaitSemaphores[%u] (%s) is a timeline semaphore, but pSubmits[%u] does not"
```
Comment (filnet) [inline layers/core_validation.cpp:2621]: missing space after "not" ?

File: layers/core_validation.cpp [hunk 2]
```diff
@@ -2616,29 +2616,32 @@ bool CoreChecks::ValidateSemaphoresForSubmit(VkQueue queue, const VkSubmitInfo *
...
         if (pSemaphore && pSemaphore->type == VK_SEMAPHORE_TYPE_TIMELINE_KHR && timeline_semaphore_submit_info &&
             submit->waitSemaphoreCount != timeline_semaphore_submit_info->waitSemaphoreValueCount) {
-            skip |= LogError(semaphore, "VUID-VkSubmitInfo-pNext-03240",
-                             "VkQueueSubmit: %s is a timeline semaphore, it contains an instance of"
-                             "VkTimelineSemaphoreSubmitInfoKHR, but waitSemaphoreValueCount is different than "
-                             "waitSemaphoreCount",
-                             report_data->FormatHandle(semaphore).c_str());
+            skip |=
+                LogError(semaphore, "VUID-VkSubmitInfo-pNext-03240",
+                         "VkQueueSubmit: pSubmits[%u].pWaitSemaphores[%u] (%s) is a timeline semaphore, it contains an instance of"
```
Comment (filnet) [inline layers/core_validation.cpp:2629]: missing space after "of" ?

Comment (sfricke-samsung) [reply layers/core_validation.cpp:2629]: no you are correct, this is a good catch... usually, when I create a new error message clang-format reformats it for me, but appending to it, I didn't account for the space... will fix all places I messed this up

Comment (sfricke-samsung) [reply layers/core_validation.cpp]: fixed... just made each string a single line, that way clang-format could do it's job properly and decide when to line break

**Comment by mark-lunarg** [comment]
Running through CI.

**Pull request merged**